libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/market-roles/dh-market-participant-actor-market-roles.component.html

```diff
@@ -0,0 +1,30 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let t; read: 'marketParticipant.organization'">
+  <mat-selection-list
+    (selectionChange)="onSelectionChange()"
+    [(ngModel)]="listModel"
+  >
+    <mat-list-option
+      *ngFor="let marketRole of availableMarketRoles"
+      [value]="marketRole"
+      [disabled]="invalidInCurrentSelection(marketRole)"
+    >
+      {{ t("marketRoles." + marketRole) }}
```

Comment:
suggestion: looks like `marketParticipant.organization.marketRoles` translation key is used both here and in "dh-market-participant-organization-overview.component.html#65". Let's move the `marketRoles` object (in both translation files) one level up so it's nested right under `marketParticipant`. 

This way the `read` scope here (line 17) will become `marketParticipant.marketRoles` and the translation output itself will be `{{ t(marketRole) }}`.

Then in "dh-market-participant-organization-overview.component.html#64-66" we'll have:

```
<div *ngFor="let mr of element.actor.marketRoles">
  {{ "marketParticipant.marketRoles." + mr.eicFunction | transloco }}
</div>
```